test/integration/cookie.spec.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-underscore-dangle */
+const helper = require('./support/integrationSpecHelper');
+const googleTagManagerHelper = helper.googleTagManagerHelper;
+const cookiePage = helper.cookiePage;
+const expect = require('chai').expect;
+
+describe('Cookie page', () => {
+  before(() =>
+    cookiePage.visit()
+  );
+
+  it('should contain valid googleTagManager', () =>
+    expect(googleTagManagerHelper.getUserVariable()).to.exists
+  );
+
+  it('displays govuk general cookie info', () =>
+    expect(cookiePage.isDisplayed()).to.equal(true)
+  );
+});
```

Comment:
This is the actual cookie page as opposed to the drop down. See https://tools-stage.universal-credit.service.gov.uk/work-you-could-do/3449ae9e-24be-445e-8e42-db0c833f5bec/cookies